easypy/logging/__init__.py

```diff
@@ -145,7 +145,10 @@ def indented(self, header=None, *args, level=AUTO, timing=True, footer=True):
         from easypy.timing import timing as timing_context
 
         level = if_auto(level, G.NOTICE)
-        header = (header % args) if header else ""
+        if header:
+            header = (header % args) if args else header
+        else:
+            header = ""
```

Comment:
choose an `if` styles and stick to it:

```suggestion
        if not header:
            header = ''
        elif args:
            header = (header % args)
```
or
```suggestion
        header = '' if not header else (header % args) if args else header
```

(the 1st is better, since having `header = .... header` is awkward)